src/app/util/attribute-storage.cpp

```diff
@@ -261,9 +261,43 @@ uint16_t emberAfGetDynamicIndexFromEndpoint(EndpointId id)
     return kEmberInvalidEndpointIndex;
 }
 
+const EmberAfCluster * getClusterTypeDefinition(EndpointId endpointId, ClusterId clusterId, EmberAfClusterMask mask)
```

Comment:
this should be in an anonymous namespace sicne we do not export it (if it is intended for external API it should have the `emberAf` prefix. However we probably do not want to extend the API surface.

Reply:
done